src/Engine/ThirdParty/CoherentLabs_rpmalloc/rpmalloc.cpp

```diff
@@ -11,6 +11,8 @@
 *
 */
 
+#ifdef USE_RPMALLOC
```

Comment:
see comment below

Reply:
I am unable to take this out without the compiler returning the following:

rpmalloc 366: error: thread-local storage is not supported for the current target
RPMALLOC_DEFINE_THREAD_LOCAL(uint32_t, _memory_preferred_heap)